velox/functions/lib/DateTimeFormatter.cpp

```diff
@@ -189,28 +190,40 @@ constexpr int monthsFullLength[] = {7, 8, 5, 5, 3, 4, 4, 6, 9, 7, 8, 8};
 /// \param padFront if the padding is in front of the content or back of the
 /// content.
 template <typename T>
-std::string padContent(
+void padContent(
     const T& content,
-    char padding,
-    size_t totalDigits,
-    bool padFront = true) {
+    const char padding,
+    const size_t totalDigits,
+    common::OutputByteStream& outStream,
+    const bool padFront = true) {
   std::string strContent = folly::to<std::string>(content);
   auto contentLength = strContent.size();
   if (contentLength == 0) {
-    return std::string(totalDigits, padding);
+    outStream.appendMultiple(padding, totalDigits);
+    return;
   }
 
   bool isNegative = strContent[0] == '-';
   auto digitLength = contentLength - (isNegative ? 1 : 0);
   if (digitLength >= totalDigits) {
-    return strContent;
+    outStream.append(strContent.data(), strContent.size());
```

Comment:
Can we eliminate intermediate string strContent?

This could use memcpy.

Reply:
It seems we have to use an intermediate string. As we need it to keep the string result of cast by folly: `const std::string strContent = folly::to<std::string>(content)`. Right?

Reply:
@PHILO-HE Can we use std::to_chars instead of folly::to? https://en.cppreference.com/w/cpp/utility/to_chars